### PR TITLE
Added Contact Us page

### PR DESCRIPTION
**Pull request by pardis-deriv**
# Summary
Create Cypress test QATEST-1651 - About Us - Contact Us

# Ticket
https://app.clickup.com/t/20696747/DPROD-3782

Screenshot 2024-03-29 at 4 54 07 PM

### CODE REVIEW

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Review by bilalmajid-deriv — APPROVED** [review]

File: cypress/e2e/full/aboutUsContactUs.cy.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    cy.get('a[href*="/maps/"]').each(($link) => {
+        cy.wrap($link).invoke('attr', 'href').should('include', '/maps/');
+    });
+
+    cy.findByRole('heading', { name: 'Business Partnership' }).should('be.visible');
+    cy.findByText('Contact us via live chat').should('be.visible');
+}
+
+describe('QATEST-1651 - should validate the About Us - Contact us', () => {
+    it('should validate Contact Us page for EU', () => {
```
Comment (bilalmajid-deriv) [inline cypress/e2e/full/aboutUsContactUs.cy.js:55]:
```suggestion
    it('should validate Contact Us page for EU', {tags: ['@full-tests', '@eu-tests']}, () => {
```

File: cypress/e2e/full/aboutUsContactUs.cy.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        cy.c_visitResponsive(`/contact-us/${Cypress.env('RegionEU')}`, 'desktop');
+        validateAboutUsContactUs();
+    });
+
+    it('should validate Contact Us page for ROW', () => {
```
Comment (bilalmajid-deriv) [inline cypress/e2e/full/aboutUsContactUs.cy.js:60]:
```suggestion
    it('should validate Contact Us page for ROW', {tags: ['@full-tests', '@row-tests']}, () => {
```

File: cypress/e2e/full/aboutUsContactUs.cy.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        cy.c_visitResponsive(`/contact-us/${Cypress.env('RegionROW')}`, 'desktop');
+        validateAboutUsContactUs();
+    });
+});
+
+describe('QATEST-1651- Responsive - should validate the About Us - Contact us', () => {
+    it('should validate Contact Us page for EU', () => {
```
Comment (bilalmajid-deriv) [inline cypress/e2e/full/aboutUsContactUs.cy.js:67]:
```suggestion
    it('should validate Contact Us page for EU', {tags: ['@full-tests', '@eu-tests']}, () => {
```

File: cypress/e2e/full/aboutUsContactUs.cy.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        cy.c_visitResponsive(`/contact-us/${Cypress.env('RegionEU')}`);
+        validateAboutUsContactUs();
+    });
+
+    it('should validate Contact Us page for ROW', () => {
```
Comment (bilalmajid-deriv) [inline cypress/e2e/full/aboutUsContactUs.cy.js:72]:
```suggestion
    it('should validate Contact Us page for ROW', {tags: ['@full-tests', '@row-tests']}, () => {
```

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]
can you please add respective tags

**Pull request closed without merging**